mathgenerator/mathgen.py

```diff
@@ -92,6 +92,25 @@ def powerRuleDifferentiationFunc(maxCoef = 10, maxExp = 10, maxTerms = 5):
         solution += str(coefficient * exponent) + "x^" + str(exponent - 1)
     return problem, solution
 
+def basicAlgebraFunc(maxVariable = 10):
+    a = random.randint(1, maxVariable)
+    b = random.randint(1, maxVariable)
+    c = random.randint(b, maxVariable)
+    # calculate gcd
+    def calculate_gcd(x, y):
+        while(y):
+            x, y = y, x % y
+        return x
+    i = calculate_gcd((c - b), a)
+    x = f"{(c - b)//i}/{a//i}"
```

Comment:
Almost perfect. I think that if the denominator of the solution is 1, the solution should just be the value of the numerator. Some cases I found were:
```
('2x + 1 = 9', '4/1')
('4x + 5 = 9', '1/1')
```
Make sure you test your code before you pr. You can do this by adding `print(basicAlgebra())` or whatever class instance you want to test at the bottom of your code and then running it. You can also do
```
for x in range(0, 5):
    print(basicAlgebra())
```
to test out your code multiple times.